src/components/Requirements/SubRequirement.vue

```diff
@@ -572,6 +572,7 @@ button.view {
   &-name {
     text-align: left;
     margin-left: 11px;
+    max-width: 12rem;
```

Comment:
Nit: Could we do `max-width: 11rem`? 

When `max-width: 12rem`, it would be better if the "15/18 credits" line for the Liberal Studies Distribution: 18 credits requirement weren't broken up into 2 lines as shown below:

![Screen Shot 2021-04-01 at 8 32 29 PM](https://user-images.githubusercontent.com/54298311/113367192-61e0d980-9329-11eb-91fd-7d2f56eed410.png)

If we make `max-width: 11rem`, then "15/18 credits" fits on one line.
![Screen Shot 2021-04-01 at 8 34 03 PM](https://user-images.githubusercontent.com/54298311/113367286-9b194980-9329-11eb-976f-4e7046348716.png)